src/Sentry/Internal/Json.cs

```diff
@@ -26,12 +26,22 @@ internal static class Json
             new StreamWriter(stream, Encoding, 1024, true)
         );
 
+        private static JsonTextReader CreateReader(Stream stream) => new JsonTextReader(
+            new StreamReader(stream, Encoding, false, 1024, true)
```

Comment:
1024 is also the default on `StreamReader` when you initialize it with another overload.

As for `stackalloc`, as mentioned, it doesn't actually work in async methods (I presume since the resulting method gets compiled into a state machine, with the state stored on the heap).

Reply:
Fair about the default of the reader.

bummer with the stackalloc lets see what else we can do.